realm/realm-library/src/objectServer/java/io/realm/permissions/PermissionChange.java

```diff
@@ -18,6 +18,7 @@
 import java.util.Date;
 import java.util.UUID;
 
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
```

Comment:
http://stackoverflow.com/questions/37422191/how-to-remove-suppressfbwarnings-annotations-from-released-jar-s doesn't sound all that reassuring to me. This should be easy to test with one of out sample projects. Perhaps it would be an idea to let the `introExample` project crash on warnings. That will also ensure we don't accidentally introduce new warnings (which we currently don't verify AFAIK).